### PR TITLE
Course 13

### PR DESCRIPTION
**Pull request by joshua-alex**
Adding course 13 notes and homework.

### CODE REVIEW

File: Course 13 - GitHub!/Course13Notes.js [hunk 1]
```diff
@@ -1,52 +1,97 @@
...
-Creating a new repository and connecting it to github
-1. Navigate to a folder you want that you want to make into a github repo
-2. git init - to initialize the git process and create a brand new repo
-3. git add . - to add the files to a staging area WHAT GOOD IS THIS FOR???
-4. git commit -m "<notes>" - to commit the changes to SOMET?HING!!
-5. Log in to your github account and click on new repository and create repository
-6. You should see some instructions there, which we will follow
-7. git remote add origin https://github.com/username/new_repo - WHAT DOES THIS DO SPECIFICALLY?
-8. git push -u origin master - this pushes your local repo to your github account
-9. Check your github account and see that your work has been pushed to github
```
Comment (znegomir) [inline Course 13 - GitHub!/Course13Notes.js]: Leave this in

File: Course 13 - GitHub!/Course13Notes.js [hunk 2]
```diff
@@ -1,52 +1,97 @@
...
+    -this commits your changes to your local branch
 
-git commit - adds your added changes to something else lol? local branch?
-git commit -m "Message here" - commits changes with a note about the changes
+11. git push
+    -pushes your changes to the remote branch 
+    -can also use git push origin <remote branch name>
 
-git push - pushes commited changes to remote branch
-git push -u - maybe??? why???
```
Comment (znegomir) [inline Course 13 - GitHub!/Course13Notes.js]: Review the syntax for this command and update accordingly

File: Course 13 - GitHub!/Course13Notes.js (hunk 1 above)
Comment (joshua-alex) [reply Course 13 - GitHub!/Course13Notes.js]: @znegomir I can add this back into Course 13 if you think it's useful.

File: Course 13 - GitHub!/Course13Notes.js (hunk 2 above)
Comment (joshua-alex) [reply Course 13 - GitHub!/Course13Notes.js]: @znegomir The "lols" and others were removed with the latest commit. The [files changed](https://github.com/partnerhero/Programming-101/pull/9/files) has some filters you can use to show changes from "all commits"

**Pull request merged**